charts/thanos/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: thanos
 description: Helm chart to install Thanos; a set of components that can be composed into a highly available metric system with unlimited storage capacity, which can be added seamlessly on top of existing Prometheus deployments.
 type: application
-version: 1.21.1
+version: 1.21.2
```

```suggestion
version: 1.21.1
```
Please don't modify the chart file.